starlite/middleware/logging.py

```diff
@@ -0,0 +1,79 @@
+import logging
+from typing import TYPE_CHECKING
+
+from starlite.connection import Request
+from starlite.enums import ScopeType
+from starlite.middleware.base import MiddlewareProtocol
+
+if TYPE_CHECKING:
+    from starlette.types import ASGIApp, Message, Receive, Scope, Send
+
+    from starlite.logging import LoggingConfig
+
+
+logger = logging.getLogger(__name__)
+
```

Comment:
The only thing I think my proposed code from #415 would change is the need to add a line here.
```python
logger.handlers[0].start_listener()
```